yarn.lock

```diff
@@ -2392,12 +2392,22 @@
     maplibre-gl "^2.1.9"
     react-map-gl "^7.0.15"
 
-"@opentripplanner/building-blocks@^1.0.3":
-  version "1.0.3"
-  resolved "https://registry.yarnpkg.com/@opentripplanner/building-blocks/-/building-blocks-1.0.3.tgz#74de32daba0d7fae7d10d5738db6d2f2824da3dd"
-  integrity sha512-065pZZTzR+qisgbvJd9swhitMMin1/ooMwuzqqRknsKYK+wOGge3Y3sbgQZATyjR/oH43uF0P2ZhGcbgqgM+YQ==
+"@opentripplanner/base-map@^3.2.0":
```

Comment:
I'm a little confused about the diff in this yarn.lock, because I don't understand why it would be adding so many new lines if it's updating minor package versions? @binh-dam-ibigroup does this seem weird to you? At the very least maybe some of these can be deduplicated.

Reply:
@josh-willis-arcadis Can you deduplicate the package references? Run `yarn-deduplicate`.